src/test/java/org/sagebionetworks/web/unitclient/widget/entity/MarkdownWidgetTest.java

```diff
@@ -0,0 +1,173 @@
+package org.sagebionetworks.web.unitclient.widget.entity;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyBoolean;
+import static org.mockito.Matchers.anyMap;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.sagebionetworks.repo.model.wiki.WikiPage;
+import org.sagebionetworks.web.client.GWTWrapper;
+import org.sagebionetworks.web.client.PortalGinInjector;
+import org.sagebionetworks.web.client.SynapseClientAsync;
+import org.sagebionetworks.web.client.SynapseJSNIUtils;
+import org.sagebionetworks.web.client.cookie.CookieProvider;
+import org.sagebionetworks.web.client.resources.ResourceLoader;
+import org.sagebionetworks.web.client.resources.WebResource;
+import org.sagebionetworks.web.client.utils.Callback;
+import org.sagebionetworks.web.client.widget.WidgetRendererPresenter;
+import org.sagebionetworks.web.client.widget.entity.ElementWrapper;
+import org.sagebionetworks.web.client.widget.entity.MarkdownWidget;
+import org.sagebionetworks.web.client.widget.entity.MarkdownWidgetView;
+import org.sagebionetworks.web.client.widget.entity.controller.SynapseAlert;
+import org.sagebionetworks.web.client.widget.entity.registration.WidgetRegistrar;
+import org.sagebionetworks.web.shared.WidgetConstants;
+import org.sagebionetworks.web.shared.WikiPageKey;
+import org.sagebionetworks.web.test.helper.AsyncMockStubber;
+
+import com.google.gwt.junit.GWTMockUtilities;
+import com.google.gwt.user.client.rpc.AsyncCallback;
+import com.google.gwt.user.client.ui.Widget;
+
+public class MarkdownWidgetTest {
+
+	MarkdownWidget presenter;
+	CookieProvider mockCookies;
+	PortalGinInjector mockInjector;
+	GWTWrapper mockGwt;
+	SynapseClientAsync mockSynapseClient;
+	SynapseJSNIUtils mockSynapseJSNIUtils;
+	MarkdownWidgetView mockView;
+	WidgetRegistrar mockWidgetRegistrar;
+	ResourceLoader mockResourceLoader;
+	SynapseAlert mockSynAlert;
+	WikiPageKey mockWikiPageKey;
+	WikiPage mockWikiPage;
+	ElementWrapper mockElementWrapper;
+	WidgetRendererPresenter mockWidgetRendererPresenter;
+	
+	String testMarkdown = "markdown";
+	String elementContentType = "image";
+	Exception caught = new Exception("test");
+	
+	@Before
+	public void setup() {
+		GWTMockUtilities.disarm();
```

Comment:
Do you still need to disarm/restore GWT.create now that Element has been wrapped?